recipes/csbschema/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "csbschema" %}
+{% set version = "1.1.2" %}
+{% set python_min = "3.8" %}
```

Comment:
conda-forge doesn't support python<3.9, so you can just drop this line and use the global pinning to `python_min = "3.9"` (it gets pulled in implicitly).

Reply:
Done.